po/da.po

```diff
@@ -3601,3797 +3633,3507 @@ msgstr ""
-msgid "Heap: Storage of values outside of function calls."
+#: src/basic-syntax/static-and-const.md:49
+msgid ""
+"Because `static` variables are accessible from any thread, they must be "
+"`Sync`. Interior mutability is possible through a [`Mutex`](https://doc.rust-"
+"lang.org/std/sync/struct.Mutex.html), atomic or similar. It is also possible "
+"to have mutable statics, but they require manual synchronisation so any "
+"access to them requires `unsafe` code. We will look at [mutable statics](../"
+"unsafe/mutable-static-variables.md) in the chapter on Unsafe Rust."
 msgstr ""
 
-#: src/memory-management/stack-vs-heap.md:10
-msgid "Values have dynamic sizes determined at runtime."
+#: src/basic-syntax/static-and-const.md:57
+msgid "Mention that `const` behaves semantically similar to C++'s `constexpr`."
 msgstr ""
 
-#: src/memory-management/stack-vs-heap.md:11
-msgid "Slightly slower than the stack: some book-keeping needed."
+#: src/basic-syntax/static-and-const.md:58
+msgid ""
+"`static`, on the other hand, is much more similar to a `const` or mutable "
+"global variable in C++."
 msgstr ""
 
-#: src/memory-management/stack-vs-heap.md:12
-msgid "No guarantee of memory locality."
+#: src/basic-syntax/static-and-const.md:59
+msgid ""
+"`static` provides object identity: an address in memory and state as "
+"required by types with interior mutability such as `Mutex<T>`."
 msgstr ""
 
-#: src/memory-management/stack.md:1
-msgid "Stack and Heap Example"
-msgstr "Eksempel på stak og heap"
-
-#: src/memory-management/stack.md:3
+#: src/basic-syntax/static-and-const.md:60
 msgid ""
-"Creating a `String` puts fixed-sized metadata on the stack and dynamically "
-"sized data, the actual string, on the heap:"
+"It isn't super common that one would need a runtime evaluated constant, but "
+"it is helpful and safer than using a static."
 msgstr ""
 
-#: src/memory-management/stack.md:6
-msgid ""
-"```rust,editable\n"
-"fn main() {\n"
-"    let s1 = String::from(\"Hello\");\n"
-"}\n"
-"```"
+#: src/basic-syntax/static-and-const.md:61
+msgid "`thread_local` data can be created with the macro `std::thread_local`."
 msgstr ""
 
-#: src/memory-management/stack.md:12
-msgid ""
-"```bob\n"
-" Stack                             Heap\n"
-".- - - - - - - - - - - - - -.     .- - - - - - - - - - - - - - - -.\n"
-":                           :     :                               :\n"
-":    s1                     :     :                               :\n"
-":   +-----------+-------+   :     :   +----+----+----+----+----+  :\n"
-":   | ptr       |   o---+---+-----+-->| H  | e  | l  | l  | o  |  :\n"
-":   | len       |     5 |   :     :   +----+----+----+----+----+  :\n"
-":   | capacity  |     5 |   :     :                               :\n"
-":   +-----------+-------+   :     :                               :\n"
-":                           :     `- - - - - - - - - - - - - - - -'\n"
-"`- - - - - - - - - - - - - -'\n"
-"```"
```

Comment:
Hi @dyoo, this shows a (minor) problem with the skipping logic: we skip the ASCII art drawing because it doesn't contain `"` or `//`. So the logic should probably be made conditional on the programming language of the code block.